lib/bundler/stats/cli.rb

```diff
@@ -70,6 +84,20 @@ def draw_show(stats, target)
         say ""
       end
 
+      def draw_versions(stats, target)
+        say "bundle-stats for #{target}"
+        say ""
+        say "depended upon by (#{stats[:top_level_dependencies].count}):\n"
+        say "+------------------------------|-------------------+"
+        say "| Name                         | Required Version  |"
+        say "+------------------------------|-------------------+"
+        stats[:top_level_dependencies].each do |stat_line|
+          say "| %-28s | %-17s |" % [stat_line[:name], stat_line[:version]]
+        end
+        say "+------------------------------|-------------------+"
+        say ""
+      end
+
```

Comment:
Man I need to just use tableprint :) I think I built this thing without internet access, and manually outputting strings is 💩

Reply:
(note: not asking you to change it, just haven't looked at that code lately)

Reply:
Hey it works with no dependencies ;-)